src/Subscriber.ts

```diff
@@ -247,18 +247,18 @@ class SafeSubscriber<T> extends Subscriber<T> {
     }
   }
 
-  private __tryOrUnsub(fn: Function, value?: any): void {
+  private __tryOrUnsub(fn: Function, ...value: Array<any>): void {
     try {
-      fn.call(this._context, value);
+      fn.call(this._context, ...value);
```

Comment:
ah yes, what I meant of `same` is the perspective of consequences only without perf differences. again, I just picked it up for readability at the moment.